lib/extractor/index.ts

```diff
@@ -196,7 +197,14 @@ function layersWithLatestFileModifications(
 
       // not adding the original file to extractedLayers
       // and removing it from the set since it can be found in consecutive layers
-      if (removedFilesToIgnore.has(filename)) {
+      if (
+        removedFilesToIgnore.has(filename) ||
+        Array.from(removedFilesToIgnore).some((removedFile) =>
+          isFileInFolder(removedFile, filename),
+        )
+      ) {
+        // If the file is in a deleted folder, this next line won't work.
```

Comment:
Could we have a separate condition for files within deleted folders so that it is clear that we don't want to run `removedFilesToIgnore.delete(filename);` for those cases?

Reply:
I edited it to include `continue` statements instead of if-else chains